kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/AdminClientServiceImpl.java

```diff
@@ -25,9 +26,13 @@
 
   @Override
   public Mono<ReactiveAdminClient> get(KafkaCluster cluster) {
-    return Mono.justOrEmpty(adminClientCache.get(cluster.getName()))
-        .switchIfEmpty(createAdminClient(cluster))
-        .map(e -> adminClientCache.computeIfAbsent(cluster.getName(), key -> e));
+    if (cluster != null && StringUtils.hasText(cluster.getName())) {
```

Comment:
there shoult be no case when we pass invalid cluster to this method, so nonnull & name checks should not be used. Or this checks should be converted to asserts